charts/MapTab.tsx

```diff
@@ -91,16 +91,16 @@ class MapWithLegend extends React.Component<MapWithLegendProps> {
         return datakeys && datakeys.length > 0
     }
 
-    @action.bound onClick(d: GeoFeature) {
+    @action.bound onClick(d: GeoFeature, ev: React.MouseEvent<SVGElement>) {
         if (!this.isEntityClickable(d.id)) return
-
         const { chart } = this.context
-        const entity = this.props.mapToDataEntities[d.id as string]
-        const keys = chart.data.availableKeysByEntity.get(entity)
+        const entityName = this.props.mapToDataEntities[d.id as string]
 
-        if (keys && keys.length) {
+        if (!ev.shiftKey) {
             chart.tab = "chart"
-            chart.data.selectedKeys = keys
+            chart.data.selectOnlyThisEntity(entityName)
+        } else {
+            chart.data.toggleEntitySelectionStatus(entityName)
```

Comment:
I _think_ we need something like `ev.stopPropagation()` or `ev.preventDefault()` here, otherwise some chart text (the title and stuff) gets a selection for me in many cases.

Reply:
Ah, good idea. I hit that exact same thing too. I've worked around that before and will dig up some old code to see how it's done properly.

Reply:
Prevent default on mousedown on the SVG seemed to do the trick. Thanks!